### DISCUSSION TITLE
Accessorio zigbee per virtualizzare pulsanti Vimar

### DISCUSSION THREAD

**Discussion by hdmvide** [indomus/forum#2928] [GENERALE - Discussioni generiche]
Ciao a tutti, 
avrei bisogno di un consiglio domotico.

In casa mia utilizzo soltanto luci hue, Ikea o, comunque, zigbee.
Sino ad ora per controllarle (accendere e spegnere)  ho sempre dovuto usare switch a batteria dei vari produttori, lasciando inutilizzati gli interruttori di casa dell’impianto elettrico. Cioè questi ultimi sono sempre accesi in modo da alimentare le lampadine gestite in modalità smart.

Adesso vorrei poter far fuori tutti questi switch a batteria dei vari produttori e utilizzare dei pulsanti a muro che andrebbero a sostituire gli interruttori dell’impianto di casa.

Conoscete un accessorio zigbee che mi permetta di convertire un pulsante (vimar nel mio caso, ma poco importa) in un pulsante virtuale da integrare nella mia domotica?

Mi spiego meglio.
Non mi riferisco ai soliti relè, come il sonoff ZBMINI-L2 ad esempio, che ad una pressione toglie e da la corrente ad una lampadina.
Le lampadine devono continuare ad essere alimentate e ad essere accese e spente dal mio bridge Hue (o HomeAssistant nel caso).
A tal proposito mi servirebbe un accessorio zigbee (preferisco che sia zigbee) che si interponga tra neutro e fase della scatola a muro e il pulsante vimar.
In questo modo l’accessorio verrebbe alimentato con la 220v, e la sua funzione sarebbe quella di virtualizzare il pulsante a muro nella domotica. Se premo il pulsante vimar si accede una luce, se lo premo di nuovo si spegne.

Cercando in rete ho trovato il Philips Hue Wall Switch che fa esattamente quello che intendo ma, oltre ad essere costosissimo, funziona a batteria.

Esiste un’alternativa?

Avevo pensato che, volendo, potrei usare un sonoff ZBMINI-L2 con il solo fine di virtualizzare il pulsante vimar. Nel senso che in uscita non collego alcuna lampadina. Collego solo la fase dall’impianto e il pulsante vimar ad uno dei due switch del sonoffw.
In questo modo se premo il pulsante vimar si aziona il rele (ma senza avere una luce in uscita da accendere)
Avendo un rele che si azione alla pressione del pulsante, potrei creare un automazione che se il rele si attiva, allora accendo una determinata lampadina e spengo il rele acceso.

Voi che mi consigliate?

Grazie

**Comment by alber1985** [comment]
Ciao,
io avevo il tuo stesso problema. La situazione è molto fastidiosa soprattutto perchè quando hai ospiti a casa "spengono" le luci con i deviatori meccanici (anch'io come te vimar). Io ho risolto in questo modo: ho inserito uno shelly 1 per ciascun interruttore. Due cose sono importanti: 1)Avere il neutro a disposizione 2) In caso di deviatori inserire lo shelly nell'ultimo, quello cioè che va alla lampada. Un altro consiglio che ti posso dare è, se puoi di inserire gli shelly direttamente nelle cassette di derivazione: hai più spazio e se hai bisogno di fare manutenzione sono tutti lì, senza bisogno di smontare le placchette degli interruttorei. Una volta installati, puoi configurarli come DETACHED, cioè sempre ON e rilevano il movimento meccanico dell'interruttore classico come trigger per abilitare vari scenari. Se sei esperto in HOME ASSISTANT puoi anche costruire automazioni molto interessanti. E' importante che tu interponga uno shelly cosi, ti resta la possibilità di "staccare" la corrente alle lampadine se ne hai necessità (spegnendo appunto da app gli shelly). Se invece lasci dei dispositivi a batteria, oltre che avere la rottura che possono scaricarsi o non funzionare, hai le lampadine sempre sotto tensione. Situazione corretta, ma nel caso avesso la necessità di staccarle dovresti tutte le volte operare sull'impianto.

**Reply by hdmvide** [reply]
Grazie per il tuo feedback.
Onestamente stavo cercando una soluzione zigbee che facesse esattamente quello che ottieni con lo Shelly, perché ritengo più affidabile una rete mesh creata ad hoc per lo scopo piuttosto che usare il Wi-Fi che non nasce per quello scopo.

ad ogni modo, proverò la soluzione da te consigliata. Ti ringrazio

**Comment by 50494554524F** [comment]
se vuoi il pulsante tutto completo xiaomi mijia switch (bluetooth) o xiaomi aquara switch (zigbee) ... i più semplici e puliti

**Reply by hdmvide** [reply]
Grazie per il tuo intervento ma il mio obiettivo è quello di abbandonare totalmente gli switch di terze parti per usare quello dell’impianto di casa.

**Reply by 50494554524F** [reply]
"Adesso vorrei poter far fuori tutti questi switch a batteria dei vari produttori e utilizzare dei pulsanti a muro che andrebbero a sostituire gli interruttori dell’impianto di casa."

l'hai scritto te :D

**Reply by hdmvide** [reply]
Ed è esattamente come ho scritto :-)… al momento in casa ho degli interruttori vimar che voglio sostituire con dei pulsanti (sempre vimar).

A differenza degli interruttori che hanno due posizioni, i pulsanti tornano allo stato primario una volta premuto: l’obiettivo è quello di usare dei pulsanti con un dispositivo simile al sonoff o Shelly, per virtualizzare il pulsante dell’impianto di casa.

preferire un dispositivo zigbee per lo scopo. Il problema dello ZBMINI-L2 è che non supporta una detached mode.

**Comment by alber1985** [comment]
Puoi provare i fibaroIl giorno 29 feb 2024, alle ore 21:39, hdmvide ***@***.***> ha scritto:﻿
Grazie per il tuo feedback.
Onestamente stavo cercando una soluzione zigbee che facesse esattamente quello che ottieni con lo Shelly, perché ritengo più affidabile una rete mesh creata ad hoc per lo scopo piuttosto che usare il Wi-Fi che non nasce per quello scopo.
ad ogni modo, proverò la soluzione da te consigliata. Ti ringrazio

**Reply by hdmvide** [reply]
Gli do un’occhiata, grazie!!
